presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -117,6 +117,9 @@
     private final Duration maxRetryTime;
     private final int maxRetries;
 
+    private volatile boolean metastoreKnownToSupportTableParamEqualsPredicate;
```

Comment:
Maybe shorten `KnownToSupport` to `Supports`

Reply:
Then i wouldn't be able to use `false` as an initial value (because initially i don't know _yet_).